python/arcticdb/util/test.py

```diff
@@ -90,22 +100,20 @@ def random_string(length: int):
 
 def get_sample_dataframe(size=1000, seed=0):
     np.random.seed(seed)
-    df = pd.DataFrame(
-        {
-            "uint8": random_integers(size, np.uint8),
-            "strings": [random_string(10) for _ in range(size)],
-            "uint16": random_integers(size, np.uint16),
-            "uint32": random_integers(size, np.uint32),
-            "uint64": random_integers(size, np.uint64),
-            "int8": random_integers(size, np.int8),
-            "int16": random_integers(size, np.int16),
-            "int32": random_integers(size, np.int32),
-            "int64": random_integers(size, np.int64),
-            "float32": np.random.randn(size).astype(np.float32),
-            "float64": np.arange(size, dtype=np.float64),
-            "bool": np.random.randn(size) > 0,
-        }
-    )
+    df = pd.DataFrame({
```

Comment:
Please can you avoid making spurious formatting changes like this, as it increases the probability of conflicts with other changes and makes the code review harder

Reply:
this is from the black formatter, I haven't touched this part of the code

Reply:
Which version of `black` are you using and what's the exact command line?

Reply:
latest `black`, with just `black --line-length 120 python/`

Reply:
Some of the formatting was indeed fixed by using black==22.8.0 which is used by our docker image and the GHA hook